packages/manager/apps/dedicated/package.json

```
@@ -44,6 +44,7 @@
     "@ovh-ux/manager-preloader": "^1.0.0",
     "@ovh-ux/manager-server-sidebar": "^2.0.0 || ^3.0.0",
     "@ovh-ux/manager-support": "^0.6.0 || ^0.7.0 || ^1.0.0",
+    "@ovh-ux/manager-user-account": "^0.0.0",
```

```suggestion
    "@ovh-ux/manager-user-account": "^0.0.0 || ^1.0.0",
```
and module should be release with a breaking change commit